### ISSUE TITLE
option for srt output 

### ISSUE THREAD

**Issue by viktorblack** [lukaszliniewicz/Pandrator#12]
a option to have a srt output file with the audio to make videos with sub

**Comment by lukaszliniewicz** [comment]
Could you please explain what you mean exactly?

**Comment by viktorblack** [comment]
just a option that makes a subtitle file together with a audio file

**Comment by lukaszliniewicz** [comment]
So when you generate audio from a text file, you'd like to be able to get a blank video file with the speech as well as subtitles of what is said? Something like that?

**Comment by viktorblack** [comment]
yes that would perfect i don't know if this is hard to do and if there are a lot of ppl wanting this but it would be perfect for me :)

**Comment by lukaszliniewicz** [comment]
I'll try to add it soon.
